3-control-flow/15_the_cyclone_2.py

```diff
@@ -7,12 +7,12 @@
 
 if height >= 137 and credits >= 10:
   print("Enjoy the ride!")
+elif credits < 10:
+  print("You don't have enough credits to ride.")
 elif height < 137:
   if height < 100 or not with_taller_person:
     print("You're not tall enough for this ride yet.")
   elif height >= 100 and with_taller_person:
     print("Enjoy the ride!")
```

Comment:
```suggestion
if credits < 10:
  print("You don't have enough credits to ride.")
elif height >= 137 and credits >= 10:
  print("Enjoy the ride!")
elif height < 137:
  if height < 100 or not with_taller_person:
    print("You're not tall enough for this ride yet.")
  elif height >= 100 and with_taller_person:
    print("Enjoy the ride!")
```

I think we should move the `credits` condition up to the top of the control flow statement to ensure that a lack of `credits` to ride is handled before the other conditions.